lib/file-store.js

```diff
@@ -33,10 +33,41 @@ FileStore.prototype.load = function (fn) {
     if (!json || json === '') {
       return fn(null, {})
     }
+
+    let store
     try {
-      fn(null, JSON.parse(json))
+      store = JSON.parse(json)
     } catch (err) {
-      fn(err)
+      return fn(err)
     }
+
+    // Check if old format and convert if needed
+    if (!store.hasOwnProperty('lastRun') && store.hasOwnProperty('pos')) {
+      if (store.pos === 0) {
+        store.lastRun = null
+      } else {
+        if (store.pos > store.migrations.length) {
+          store.pos = store.migrations.length
+        }
+
+        store.lastRun = store.migrations[store.pos - 1].title
+      }
+
+      // In-place mutate the migrations in the array
+      store.migrations.map((migration, index) => {
```

Comment:
For consistency with the rest of the code base, use traditional javascript functions unless you explicitly need the binding.  Maybe there is some other benefits that can be discussed to using arrow functions by default, but for now consistency is more important.